engine/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/PatchEmailPayloadAggregator.java

```diff
@@ -30,10 +30,10 @@ public class PatchEmailPayloadAggregator extends AbstractEmailPayloadAggregator
     private static final String NEW_ADVISORIES_EVENT = "new-advisory";
     private static final List<String> EVENT_TYPES = Arrays.asList(NEW_ADVISORIES_EVENT);
 
-    private static final String ENHANCEMENT_TYPE = "enhancement";
-    private static final String BUGFIX_TYPE = "bugfix";
-    private static final String SECURITY_TYPE = "security";
-    private static final String UNSPECIFIED_TYPE = "unspecified";
+    private static final String ENHANCEMENT_TYPE = "Enhancement";
+    private static final String BUGFIX_TYPE = "Bugfix";
+    private static final String SECURITY_TYPE = "Security";
+    private static final String UNSPECIFIED_TYPE = "Unspecified";
```

Comment:
The comparation below is done using a lower case type - and I personally like the pattern - whatever you send is lower case and we compare with lower case string versions.

```
String advisoryType = payload.getString(ADVISORY_TYPE).toLowerCase();
// Drop on unknown type
 if (!ADVISORY_TYPES.contains(advisoryType)) {
    return;
}
```

Was this failing?

Reply:
You are right, thank you for the warning :).
Changing it on Patch aggregator is little bit overkill, let's do that using simple css because it's a rendering issue.